resolwe_bio/processes/jbrowse/variants.yml

```diff
@@ -0,0 +1,39 @@
+# =========================
+# JBrowse tracks - Variants
+# =========================
+#
+---
+
+- slug: jbrowse-vcf-track
```

Comment:
The vast majority of VCFs will be produced within pipelines, so there is no major downside to having a special track, since this can be automated in workflows. The issue is if one would upload VCF directly and would need to remember to make this additional track before being able to visualize. For only this cases there is a decision of practicality  vs. efficiency to be made.